### DISCUSSION TITLE
能不能取消开启强制更新的弹框？这类弹框没有取消按钮

### DISCUSSION THREAD

**Discussion by Jiangzqts** [gkd-kit/gkd#735] [Q&A]
问题同标题

**Comment by lisonge** [comment]
你说的是 gkd 的哪个界面的哪个功能？

需要给出截图看一下

另外你说的什么强制更新？

gkd 有这个功能吗？

**Reply by Jiangzqts** [reply]
抱歉是我没描述清楚，我说的不是gkd的功能，我想咨询gkd能通过配置规则自动关闭广告，能不能通过配置规则也关闭第三方app强制更新的操作，这种强制更新升级没有取消按钮，gdk能通过配置规则禁止这种强制更新的弹窗吗？

**Reply by lisonge** [reply]
gkd 只是根据界面信息模拟用户点击，不能实现你说的功能

你说的这种需要 xposed 相关的工具才能实现，而且很容易出 bug 导致软件崩溃
